backend/graphql/types/testType.ts

```diff
@@ -10,6 +10,18 @@ const testType = gql`
     SHORT_ANSWER
   }
 
+  enum AssessmentType {
+    Beginning
+    End
```

Comment:
nit: can we make the values all caps to be consistent with the `QuestionComponentTypeEnum`? i.e `Beginning` -> `BEGINNING`

Reply:
There are places in the frontend that use Beginning and End. Should I change those as well?

Reply:
yes please! thanks 😄